examples/item-naip.json

```diff
@@ -43,65 +43,107 @@
     "datetime": "2016-09-28T00:00:00+00:00",
     "mission": "NAIP",
     "platform": "UNKNOWN_PLATFORM",
-    "gsd": 1
+    "gsd": 1,
+    "storage:schemes": {
+      "az-wus2-arc": {
+        "platform": "AZURE",
+        "region": "westus2",
+        "tier": "archive"
+      },
+      "gs-cld": {
+        "platform": "GCP",
+        "region": "us-central1",
+        "requester_pays": true,
+        "tier": "COLDLINE"
+      },
+      "aws-std": {
+        "platform": "AWS",
+        "region": "us-west-2",
+        "requester_pays": true,
+        "tier": "Standard"
+      },
+      "az-weu-hot": {
+        "platform": "AZURE",
+        "region": "westeurope",
+        "requester_pays": false,
+        "tier": "hot"
+      },
+      "az-eus-hot": {
+        "platform": "AZURE",
+        "region": "eastus",
+        "requester_pays": false,
+        "tier": "hot",
+        "deprecated": true
+      },
+      "minio": {
+        "platform": "https://play.min.io:9000"
+      }
+    }
   },
   "assets": {
+    "CO_GEOTIFF_RGB": {
+      "href": "s3://mybucket/tx/2016/100cm/rgb/30097/m_3009743_sw_14_1_20160928.tif",
```

Comment:
I don't know if it's appropriate to use an S3 URL with minio

Reply:
Works as long as you specify the endpoint URL (i.e. here `platform`) in the CLI.